src/api_client/api_resource.js

```diff
@@ -34,31 +35,29 @@ export default class APIResource {
 		return `${this.apiClient.currentNode}/api${this.path}`;
 	}
 
-	request(req, retry) {
+	request(req, retry, retryCount = 1) {
```

Comment:
Default value for `retry` should be `true`?

Reply:
I think it should be false, we probably should decide which API should retry